src/App.tsx

```diff
@@ -393,6 +417,9 @@ function App() {
           <MapIcon iconName='phone' iconImageSource='/img/phone.png' />
         </Map>
       </div>
+      <div>
```

Comment:
Is this wrapping div necessary, since there is already a div being spit out by the component that wraps the anchors?

Reply:
It's not necessary removed it.